tests/rust/package_version/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#[repr(C)]
+pub struct Foo {
+    #[cfg(not(feature = "cbindgen"))]
```

Comment:
This seems orthogonal to the feature you're testing?

Reply:
Yes, actually I could remove the whole stub code but in my latest commit, i removed just this line.